CHANGELOG.md

```diff
@@ -3,23 +3,63 @@
 ## v2.0.0-preview.1
 ### Wednesday, January 23, 2019
 
+#### Preview builds of the PowerShell extension are now available in VSCode
+
+We are excited to announce the PowerShell Preview extension in the VSCode marketplace!
+The PowerShell Preview extension allows users on Windows PowerShell 5.1 and newer to get and test the latest updates to the PowerShell extension and comes with some exciting features.
```

Comment:
May need to put Windows PowerShell 5.1 and PowerShell 6.1 and up

Reply:
Just to make sure everyone knows "it works on Core" (but I've omitted Core above because of personal brand preference... I dunno!)

Reply:
done